vector/src/main/java/im/vector/app/core/utils/SystemUtils.kt

```diff
@@ -135,9 +135,9 @@ fun startAddGoogleAccountIntent(context: Context, activityResultLauncher: Activi
 
 @RequiresApi(Build.VERSION_CODES.O)
 fun startInstallFromSourceIntent(context: Context, activityResultLauncher: ActivityResultLauncher<Intent>) {
+    val intent = Intent(Settings.ACTION_MANAGE_UNKNOWN_APP_SOURCES)
+            .setData(Uri.parse(String.format("package:%s", context.packageName)))
```

Comment:
super nit: this was already here but it could be nice to avoid `String.format` in favour of a more kotlin idiomatic `"package:${context.packageName}" `

For this usecase it's not very relevant but`String.format` is also quite heavy relative to a `StringBuilder` https://cowtowncoder.medium.com/measuring-performance-of-java-string-format-or-lack-thereof-2e1c6a13362c

Reply:
https://github.com/vector-im/element-android/issues/4859